### PR TITLE
update default config changing parallel execution to sequential execu…

### PR DESCRIPTION
**Pull request by wnbts**
*Issue #, if available:* https://github.com/aws/random-cut-forest-by-aws/issues/29

*Description of changes:* This pr changes default execution config from parallel to sequential.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jotok** [comment]
We also need to update the default parameters table in the README. Can you also add something like the following to the description of the parallelExecutionEnabled parameter: "For larger shingle sizes, dimensions, and number of trees, parallelization may improve throughput. We recommend users benchmark against their target use case."

**Comment by wnbts** [comment]
I updated the readme documentation there.

File: core/src/test/java/com/amazon/randomcutforest/RandomCutForestBuilderTest.java [hunk 1]
```diff
@@ -73,8 +73,8 @@ public void testDefaultForestWithDimensionArgument() {
         assertEquals(256 / DEFAULT_OUTPUT_AFTER_FRACTION, f.getOutputAfter());
         assertFalse(f.storeSequenceIndexesEnabled());
         assertFalse(f.centerOfMassEnabled());
-        assertTrue(f.parallelExecutionEnabled());
-        assertEquals(Runtime.getRuntime().availableProcessors() - 1, f.getThreadPoolSize());
```
Comment (jotok) [inline core/src/test/java/com/amazon/randomcutforest/RandomCutForestBuilderTest.java:77]: Can we add a new test to verify that thread pool size is being set correctly when parallel execution is set to true but no explicit value is given for thread pool size?

Comment (wnbts) [reply core/src/test/java/com/amazon/randomcutforest/RandomCutForestBuilderTest.java:77]: yes, i just added back this test case named testForestBuilderWithDefaultParallelExecutionThreadPoolSize .

**Pull request merged**